easybuild/easyconfigs/c/cURL/cURL-7.69.0-GCCcore-9.3.0.eb

```
@@ -0,0 +1,49 @@
+easyblock = 'ConfigureMake'
+
+name = 'cURL'
+version = '7.69.0'
+
+homepage = 'https://curl.haxx.se'
+
+description = """
+ libcurl is a free and easy-to-use client-side URL transfer library,
+ supporting DICT, FILE, FTP, FTPS, Gopher, HTTP, HTTPS, IMAP, IMAPS, LDAP,
+ LDAPS, POP3, POP3S, RTMP, RTSP, SCP, SFTP, SMTP, SMTPS, Telnet and TFTP.
+ libcurl supports SSL certificates, HTTP POST, HTTP PUT, FTP uploading, HTTP
+ form based upload, proxies, cookies, user+password authentication (Basic,
+ Digest, NTLM, Negotiate, Kerberos), file transfer resume, http proxy tunneling
+ and more.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+
+source_urls = ['https://curl.haxx.se/download/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['cbd638a9a4219459977da2bc7a0130ba9dfc7dc2ea3703554b47bb885ab48d83']
+
+builddependencies = [
+    ('binutils', '2.34'),
+]
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    # OS dependency should be preferred if the os version is more recent then this version,
+    # it's nice to have an up to date openssl for security reasons
+    # ('OpenSSL', '1.1.1d')
+]
+
+osdependencies = [
+    ('openssl-devel', 'libssl-dev', 'libopenssl-devel'),
+]
+
+configopts = '--with-zlib'
+# configopts += '--with-ssl=$EBROOTOPENSSL'
+
+modextravars = {'CURL_INCLUDES': '%(installdir)s/include'}
+
+sanity_check_paths = {
+    'files': ['bin/curl', 'lib/libcurl.a', 'lib/libcurl.%s' % SHLIB_EXT],
+    'dirs': ['lib/pkgconfig'],
```

The sanity check of the header files is missing
```suggestion
    'dirs': ['lib/pkgconfig', 'include/curl'],
```